candi/bashible/common-steps/node-group/002_add_system_proxy.sh.tpl

```diff
@@ -59,3 +54,5 @@ if [ -f /etc/profile.d/d8-system-proxy.sh ]; then
   rm -f /etc/profile.d/d8-system-proxy.sh
 fi
 {{- end }}
+
+bb-unset-proxy
```

Comment:
we set proxy in `if` https://github.com/deckhouse/deckhouse/pull/10676/files#diff-afcfa164cc4bcb9c363245c9f9241891aa9fc1a747b2d5bf6883009240a76931R22  above but unset always. Is it mistake?